src/electron/proc/index.ts

```diff
@@ -59,7 +59,9 @@ export class Electron {
       // electron starter
       const entry = path.join(__dirname, '../main/index');
       const args = [ entry ];
-
+      if (process.env.JEST_ELECTRON_NO_SANDBOX && process.platform !== 'win32'){
+        args.splice(0, 0, '--no-sandbox');
+      };
```

Comment:
only valid on window?

Reply:
argument `--no-sandbox` is also available on windows, but i think there is no need to disable sandbox on windows, `vscode` and `cypress` did the same choice

Reply:
so we can remove the code:
```js
process.platform !== 'win32'
```

Reply:
done. PTAL